WordPress/src/main/java/org/wordpress/android/ui/people/RoleChangeDialogFragment.java

```diff
@@ -23,6 +23,15 @@
     private RoleListAdapter mRoleListAdapter;
     private OnChangeListener mListener;
 
+    @Override
+    public void onSaveInstanceState(Bundle outState) {
+        super.onSaveInstanceState(outState);
+        String role = mRoleListAdapter.getSelectedRole();
+        if (role != null) {
```

Comment:
Being extra careful doesn't hurt but there is actually no need to check for null at this point. For simplicity's sake, I'd recommend removing the check as there is no semantic value in it.

I would even go one step further and say that you can also remove the null-check around line 136 (`radioButton.setChecked(mSelectedRole.equalsIgnoreCase(role)`) by reversing the `equals` to `role.equalsIgnoreCase(mSelectedRole)` as the `role` is never expected to be null. Cheers!